### ISSUE TITLE
Templater affects height of table cells, even if note has no templater syntax

### ISSUE THREAD

**Issue by chrisgrieser** [SilentVoid13/Templater#1307]
**Plugin information (please complete the following information):**
```
Obsidian 1.5.3
macOS 14.1 (arm)
Templater 2.1.1
Templater settings: Templater Syntax Highlighting enabled.
```

**Describe the bug**
Templater affects height of table cells. (even if the note has no templater syntax)

**Expected behavior**
table cell height should not be affected by templater

https://github.com/SilentVoid13/Templater/assets/73286100/ba8c4dff-de1d-4ff6-bbc8-d885a638727f

**Additional Context**
it's not templater's `styles.css` – I tried deleting it and restarting obsidian, the issue still occurs. Issue oily occurs with the Templater Syntax Highlighting setting enabled.

**Comment by Zachatoo** [comment]
Interesting, it looks like with Templater enabled, the list items in the table are being parsed as list items. The `- text` becomes `* text`. Whereas with Templater disabled, Obsidian doesn't try to parse them as list items. I would expect that the way we're overlaying Templater's syntax highlighting on top of Obsidian's hypermd that this wouldn't be happening.

@iamrecursion do you have any ideas on what would be causing this?

**Comment by iamrecursion** [comment]
@chrisgrieser Any chance of a minimal reproducer file for this? I suspect some strange interaction with the new table editor in LP, but I never use LP.

**Comment by chrisgrieser** [comment]
sure, here is simply (part of) the table in the video
```md
| Dimension | Accelerating Factors | Decelerating Side Effects | Limiting Factors |
| ---- | ---- | ---- | ---- |
| Speed of Individual Research Activities | - increased availability of resources  <br>- parallelization of normally sequential tasks | - less time for research due to the<br>increased need to<br>disprove low-quality<br>research | - eigentime of research objects  <br>- operational speed of methods/devices |
| Number of Concurrent Research Activities | - researchers (temporarily) switching research subject | - higher difficulty of assessing the research of others | - availability of the research object<br>- need to acquire knowledge/equipment for new field <br>- epistemic distance to research subject |
```

**Comment by iamrecursion** [comment]
Yeah I have _no_ ideas on this one so far. Is it repro with just templater and LP?

**Comment by chrisgrieser** [comment]
yep, can reproduce in the sandbox vault with nothing but templater and the table above.

**Comment by iamrecursion** [comment]
Cool, that matches my experience as well. I'm still pretty clueless on this one though. Without a templater tag activation nothing should change, and indeed none of the branches activate. I can only assume it's a bug in the overlay functionality so far.
